### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by nstepien**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,88 +1,111 @@
...
+  - ⚠️ `onGridRowsUpdated` to `onRowsUpdate`
+  - ⚠️ `rowData` to `row`
+  - ⚠️ `fromRowData` to `fromRow`
+- **Changed:**
+  - Only visible headers cells are now rendered. [#1837](https://github.com/adazzle/react-data-grid/pull/1837)
+  - ⚠️ `column.cellClass` does not affect header cells anymore.
+
+## `master` to `alpha`
+- **Added:**
+  - TypeScript declaration files
```
Comment (nstepien) [inline CHANGELOG.md]: We missed this important part before 👀

**Comment by nstepien** [comment]
I've refactored it a bit: 1 line per pertinent change, and additional info as sub-points, so it's easier to scan through the change list.

**Comment by nstepien** [comment]
Also added a ⚠ for breaking changes, also to make it easier to spot them.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,88 +1,111 @@
...
+  - `column.cellClass(row)` function support:
+    - `column = { ..., cellClass(row) { return string; } }`
+  - `scrollToRow` method
+    - ⚠️ This replaces the `scrollToRowIndex` prop
+- **Removed:**
+  - **Packages:**
+    - ⚠️ `react-data-grid-addons`
+  - **Props:**
+    - ⚠️ `cellContentRenderer`
+    - ⚠️ `contextMenu`
```
Comment (amanmahajan7) [inline CHANGELOG.md]: May be point to the `contextMenu` story on how to implement one

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,88 +1,111 @@
...
+    - ⚠️ `onCellSelected`
+    - ⚠️ `onGridKeyDown`
+    - ⚠️ `onGridKeyUp`
+    - ⚠️ `onRowDoubleClick`
+    - ⚠️ `rowsContainer`
+    - ⚠️ Subrow props: `getSubRowDetails`, `onCellExpand`, `onDeleteSubRow`, and `onAddSubRow`
+      - Check [#1853](https://github.com/adazzle/react-data-grid/pull/1853) on how to migrate
+    - ⚠️ `cellMetaData` (from `Row` and `Cell` props)
+    - ⚠️ `value` (from `column.formatter` props)
+  - ⚠️ React elements are no longer supported, please use components instead.
```
Comment (amanmahajan7) [inline CHANGELOG.md]: May we can mention `renderers` and `formatters` as an example

**Pull request merged**